interface/billing/billing_report.php

```diff
@@ -413,10 +413,10 @@ function MarkAsCleared(Type)
 else
 {
 
-$ThisPageSearchCriteriaDisplayMaster= array( xl("Date of Service"),xl("Date of Entry"),xl("Date of Billing"),xl("Claim Type"),xl("Patient Name"),xl("Patient Id"),xl("Insurance Company"),xl("Encounter"),xl("Whether Insured"),xl("Charge Coded"),xl("Billing Status"),xl("Authorization Status"),xl("Last Level Billed"),xl("X12 Partner") );
+$ThisPageSearchCriteriaDisplayMaster= array( xl("Date of Service"),xl("Date of Entry"),xl("Date of Billing"),xl("Claim Type"),xl("Patient Name"),xl("Patient Id"),xl("Insurance Company (Billed Only!)"),xl("Encounter"),xl("Whether Insured"),xl("Charge Coded"),xl("Billing Status"),xl("Authorization Status"),xl("Last Level Billed"),xl("X12 Partner (Billed Only!)"),xl("Code"),xl("Modifier") );
```

Comment:
I started cleaning this up myself in another branch, but got distracted by
something ...

https://github.com/tmccormi/openemr/tree/aethelwulffe-genericval2-billingnote-v2

Might be helpful

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo
“I am and always will be the optimist. The hoper of far-flung hopes and the
dreamer of improbable dreams.” — The Doctor, Season 6



On Mon, Feb 8, 2016 at 12:33 AM, Brady Miller <notifications@github.com>
wrote:

> In interface/billing/billing_report.php
> <https://github.com/openemr/openemr/pull/45#discussion_r52142097>:
>
> > @@ -413,10 +413,10 @@ function MarkAsCleared(Type)
> >  else
> >  {
> >
> > -$ThisPageSearchCriteriaDisplayMaster= array( xl("Date of Service"),xl("Date of Entry"),xl("Date of Billing"),xl("Claim Type"),xl("Patient Name"),xl("Patient Id"),xl("Insurance Company"),xl("Encounter"),xl("Whether Insured"),xl("Charge Coded"),xl("Billing Status"),xl("Authorization Status"),xl("Last Level Billed"),xl("X12 Partner") );
> > +$ThisPageSearchCriteriaDisplayMaster= array( xl("Date of Service"),xl("Date of Entry"),xl("Date of Billing"),xl("Claim Type"),xl("Patient Name"),xl("Patient Id"),xl("Insurance Company (Billed Only!)"),xl("Encounter"),xl("Whether Insured"),xl("Charge Coded"),xl("Billing Status"),xl("Authorization Status"),xl("Last Level Billed"),xl("X12 Partner (Billed Only!)"),xl("Code"),xl("Modifier") );
>
> agreed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openemr/openemr/pull/45/files#r52142097>.
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.